test/format/wts.c

```diff
@@ -784,13 +789,13 @@ wts_open(const char *home, WT_CONNECTION **connp, bool verify_metadata)
             CONFIG_APPEND(p, ",%s", s);
         if (g.config_open != NULL)
             CONFIG_APPEND(p, ",%s", g.config_open);
-        /*
-         * FIXME-WT-14979: Precise checkpoints are incompatible with the on-disk block manager.
-         * Ideally we would not gate precise checkpoints on disagg also being enabled.
-         */
-        if (g.disagg_storage_config && GV(CHECKPOINT_PRECISE))
+
+        if (GV(PRECISE_CHECKPOINT))
             CONFIG_APPEND(p, ",precise_checkpoint=true");
 
+        if (GV(PRESERVE_PREPARED))
```

Comment:
this is also present in create `wts_open`. Also, should we check if ops.prepared is enabled and only then we check is preserve_prepared is true?

Reply:
there will be no issue if prepared is not enabled. It is just an no op. I have to do this in both here and wts_open to ensure we pass the correct config.

Reply:
> I have to do this in both here and wts_open to ensure we pass the correct config. wts_open is called for reopen and we need to reopen with the same config.
wts_open seems to be called both here and reopen? Sorry I have misunderstood something

Reply:
I can add a comment?